src/libratbag.h

```diff
@@ -1125,6 +1141,12 @@ enum ratbag_button_type {
 	RATBAG_BUTTON_TYPE_PROFILE_DOWN,
 };
 
+struct ratbag_color {
+	unsigned short red;
+	unsigned short green;
+	unsigned short blue;
+};
```

Comment:
remember, this is the public API and it's an open-coded struct so we can't hide it behind APIs. So short needs to be all we need for... forever? if we really only care about 0-255, then unsigned char would be even better, makes it more obvious that we're only using one byte.